internal/statemachine/classic_test.go

```diff
@@ -3191,6 +3215,96 @@ func TestFailedUpdateBootloader(t *testing.T) {
 		saveCWD := helper.SaveCWD()
 		defer saveCWD()
 
+		var stateMachine ClassicStateMachine
```

Comment:
hmmm, ok, this part here is a bit confusing. This added test clearly tests the `prepareClassicImage()` and `preseedClassicImage()` workflow, but the test the code has been added to is called `TestFailedUpdateBootloader()`. I think the name needs to be adjusted?

Reply:
Oh I think git got confused while merging these branches. Silly git.